packages/airnode-node/src/evm/authorization/authorization-fetching.test.ts

```diff
@@ -166,10 +178,85 @@ describe('fetch (authorizations)', () => {
     ]);
     expect(res).toEqual({});
   });
+
+  it('does not fetch authorizations if found in config and returns true for valid authorizations', async () => {
```

Comment:
Can you also add a test for a combination of these two? Meaning that there are both requests that are authorized in the config and those that need to do the authorization on-chain.